ckanext/restricted/templates/restricted/restricted_string.html

```diff
@@ -17,33 +16,41 @@
 
   {% else %}
 
-    {% set popup_link = "<a href='#' data-module='restricted_popup' data-module-title='{}' data-module-content='{}'>{}</a>" %}
+    {% set popup_link = "<a href='#' data-module='restricted_popup' data-module-title='%s' data-module-content='%s'>%s</a>" %}
     {% set org_string = False %}
     {% set usr_string = False %}
 
     {% if restricted.get("allowed_organizations") %}
       {% set organizations = (restricted.get("allowed_organizations").split(",") + [pkg.organization.name])|unique|list %}
       {% set organizations_popup_title = _('Access granted to organizations') %}
       {% set organizations_popup_content = "<br />".join(organizations ) %}
-      {% set organizations_popup = popup_link.format(organizations_popup_title, organizations_popup_content, _('organizations')) %}
+      {% set organizations_popup = popup_link % (organizations_popup_title, organizations_popup_content, _('organizations')) %}
     {% endif %}
 
     {% if restricted.get("allowed_users") %}
       {% set users = restricted.get("allowed_users").split(",") %}
       {% set users_popup_title = _('Access granted to users') %}
       {% set users_popup_content = "<br />".join(users) %}
-      {% set users_popup = popup_link.format(users_popup_title, users_popup_content, _('users')) %}
+      {% set users_popup = popup_link % (users_popup_title, users_popup_content, _('users')) %}
     {% endif %}
     {% set x = "<b>Hello</b>"%}
 
     {% if organizations_popup and users_popup %}
-        {{_("Access restricted to specific {organizations} and {users}.").format(organizations=organizations_popup|safe, users=users_popup|safe) }}
+        {% trans organizations=organizations_popup|safe, users=users_popup|safe %}
+            Access restricted to specific {{ organizations }} and {{ users }}.
+        {% endtrans %}
     {% elif organizations_popup %}
-        {{_("Access restricted to specific {organizations}").format(organizations=organizations_popup|safe) }}
+       {% trans organizations=organizations_popup|safe %}
+           Access restricted to specific {{ organizations }}.
+       {% endtrans %}
     {% elif users_popup %}
-        {{_("Access restricted to specific {users}").format(users=users_popup|safe) }}
+        {% trans users=users_popup|safe %}
```

Comment:
This is cool! I wasn't aware you could do this. I guess this is a specific feature of `{% trans }` to allow you to define variables when initialising it